OutOfSchool/OutOfSchool.BusinessLogic/Services/CompetitiveEventService.cs

```diff
@@ -121,6 +121,55 @@ public async Task Delete(Guid id)
         }
     }
 
+    /// <inheritdoc/>
+    public async Task<SearchResult<CompetitiveEventViewCardDto>> GetByProviderId(Guid id, ExcludeIdFilter filter)
+    {
+        logger.LogDebug("Getting Competitive events by organization started. Looking ProviderId = {id}.", id);
+
+        filter ??= new ExcludeIdFilter();
+        ValidateExcludedIdFilter(filter);
+
+        var predicate = PredicateBuilder.True<CompetitiveEvent>();
+        predicate = predicate.And(x => x.OrganizerOfTheEventId == id);
+
+        if (filter.ExcludedId is not null)
+        {
+            predicate = predicate.And(x => x.Id != filter.ExcludedId);
+        }
+        var competitiveEventCardsCount = await competitiveEventRepository.Count(
+            whereExpression: predicate).ConfigureAwait(false);
+
+        var competitiveEvents = await competitiveEventRepository.Get(
+            skip: filter.From,
+            take: filter.Size,
+            includeProperties: null,
```

Comment:
extract `includeProperties` as a string readonly property, that can be empty for now. Add a `TODO` to fill it when we have final list of properties. Will be easier to modify just that string.